### DISCUSSION TITLE
Upgrade from 4.1.9 to 6.2.0

### DISCUSSION THREAD

**Discussion by DioMeo** [OxalisCommunity/Oxalis-AS4#227] [Q&A]
Hello, I have a problem with install the new version.
I started from 4.1.9 to 6.2.0, is it only a copy/paste the new version or there is any prerequisite, apart from Java version 11?
Because, if I only copy the new version I had error 404.
Could you help me please? My old colleagues did not leave any documentation.
And, oh yeah, is there a documentation for the upgrade?
Thanks.

**Comment by dladlk** [comment]
Maybe you need to start with reading release notes for both Oxalis and Oxalis-AS4 projects:
https://github.com/OxalisCommunity/Oxalis-AS4/releases
https://github.com/OxalisCommunity/oxalis/releases

Error 404 says so little about your issue, but some guesses:

- AS2 is deprecated (if you try to access /as2)
- A lot of packages were renamed from difi.no to network.oxalis 

But Oxalis is a free open-source project, which is developed for free and on "AS IS" basis, so I am not sure there is a documentation for upgrade from so old version except the source code itself... Its usage supposes deep knowledge of Peppol and some level of java programming.

**Reply by DioMeo** [reply]
Hello dladlk, thank you to reply me.
I'll write to Aaron and I hope he could support me for this problem.
